test/lru_hash_test.rb

```diff
@@ -0,0 +1,100 @@
+require 'test_helper'
```

Comment:
`minitest/hell` I believe it's called is a good way to test this file for thread-sfaety :)